lib/fortnox/api/validators/row.rb

```diff
@@ -0,0 +1,20 @@
+require "fortnox/api/validators/base"
+
+module Fortnox
+  module API
+    module Validator
+      class Row < Fortnox::API::Validator::Base
+
+        validates :article_number, size: 1..50
+        validates :description, size: 1..50
+
+        validates :account_number, inclusion: (0..9999)
+        validates :delivered_quantity, inclusion: (0..99_999_999_999_999)
+        validates :discount, inclusion: (0..999_999_999_999)
+        validates :house_work_hours_to_report, inclusion: (0..99_999)
+        validates :price, inclusion: (0..999_999_999_999)
```

Comment:
Actually, I have tried size but without any success. It appears as if size is for data types such as `String`, `Array` etc and not for `Integer`/`Float`.

If I try using it with `Float`, I get the following message:
```
undefined method `size' for 12345678901234.0:Float
```

Using it on `Integer`s does not raise any errors, but the validation tests are failing.